js/.eslintrc

```diff
@@ -169,7 +169,7 @@
         "prefer-const": 0,
         "prefer-reflect": 0,
         "prefer-spread": 2,
-        "prefer-template": 2,
+        "prefer-template": 0, // not replace content in string
```

Comment:
Leave this at `2`. I'll figure out how to exempt the tests from this particular ESLint rule.